app/models/user.rb

```diff
@@ -37,6 +37,13 @@ def mailboxer_email(_obj = nil)
     email
   end
 
+  # Override from devise to make a determination of whether to send email notification or not
+  def send_confirmation_notification?
```

Comment:
Looks like `#first_or_create` from below calls save, so calling `#skip_confirmation!` inside the create block should be enough to cover everything. I don't think we need the override for `#send_confirmation_notification?`

Reply:
We do. Margaret tested it and apparently skip_confirmation_notification! wasn't actually being called at the right time. So instead of skip_confirmation_notification! we can use #send_confirmation_notification? override to get it to make a determination.

Reply:
Alright, do you want to simplify it down to a one-liner and maybe include `#super` so it's still going through devise's checks? Something like 
`super && !(email.include?('uoregon') || email.include?('oregonstate'))`
or 
`super && !email.include?('uoregon') && !email.include?('oregonstate')`